importers/dependencies/extract_relations.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from importers.dependencies.languages import Parser
+from util.deps_util import DependencyUtils
+from util.logging_util import LoggingUtil
+import os
+
+
+class DependencyExtractor(object):
```

Comment:
Active branch of repository. May be I can add "branch_id" in "file_dependency" so that endusers can switch between branches and run. I don't think Gitana program should switch between branches. What you think?